include/circt/Dialect/RTGTest/IR/RTGTestOps.td

```diff
@@ -20,7 +20,6 @@ class RTGTestOp<string mnemonic, list<Trait> traits = []> :
 
 def CPUDeclOp : RTGTestOp<"cpu_decl", [
   Pure,
-  ConstantLike,
```

Comment:
Why is this removed?

Reply:
Operations marked with `ConstantLike` require a folder which is missing. I was wondering whether I should just add one returning the IntegerAttr or if I should add a custom attribute for that, so I took the easy way out by removing it for now and adding it later again together with the folder. But I'm also happy to add the folder right here if you prefer.

Reply:
Got, it. I was just curious so please feel free to keep as is.